src/main/java/com/bina/varsim/util/VCFWriter.java

```diff
@@ -0,0 +1,76 @@
+package com.bina.varsim.util;
+
+import com.bina.varsim.types.ChrString;
+import com.google.common.collect.ImmutableList;
+
+import java.util.List;
+import java.util.StringJoiner;
+
+/**
+ * Created by guoy28 on 3/2/18.
+ */
+public class VCFWriter {
+  /**
+   * generate VCF file header
+   *
+   * a little explanation about translocation-related meta-info lines:
+   * each locus involved in translocations is modeled independently as
+   * a cut-paste event with slight variations for different types of
+   * translocations. Basically, a region at locus A (the sink) will be
+   * cut (deleted), and a region at locus B (the source) will be placed
+   * at the sink. The placement may be: complete transfer; complete transfer
+   * with inversion; no transfer (one-way or unbalanced translocation).
+   *
+   * @param referenceFileName reference file name
+   * @param sampleNames list of sample names
+   * @return
+   */
+  public static String generateVCFHeader(final String referenceFileName, final ImmutableList<String> sampleNames) {
+    StringBuilder VCFHeader = new StringBuilder();
+    VCFHeader.append("##fileformat=VCFv4.3\n" +
```

Comment:
String template might be useful here to keep it simple. Also, you can put in the comment you have for different INFO fields in the description itself, so the VCF header is self-explanatory.